gcsio/src/test/java/com/google/cloud/hadoop/gcsio/TrackingHttpRequestInitializer.java

```diff
@@ -169,12 +171,33 @@ public ImmutableList<String> getAllRequestStrings() {
         .collect(toImmutableList());
   }
 
+  public ImmutableList<String> getAllRequestInvocationIds() {
+    return requests.stream()
+        .map(r -> getInvocationId(r.getHeaders()))
+        .filter(id -> id != null && !id.isEmpty())
```

Comment:
nit: can change this to .filter(not(Strings::isNullOrEmpty))

Reply:
Done.